examples/va/hci/control-plane.md

```diff
@@ -34,6 +34,17 @@ Wait for NNCPs to be available
 oc wait nncp -l osp/nncm-config-type=standard --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=300s
 ```
 
+## Apply remaining networking configuration
+
+Generate the reminaing networking configuration
+```
+kustomize build control-plane/networking > networking.yaml
+```
+Apply the networking CRs
+```
+oc apply -f networking.yaml
+```
+
 ## Apply networking and control-plane configuration
 
 Generate the control-plane and networking CRs.
```

Comment:
nit:
May want to change heading, remove `networking and`
May want to remove "and networking CRs"

Reply:
Ack